app/src/main/java/com/univoice/data/repositoryimpl/UniversityRepositoryImpl.kt

```diff
@@ -0,0 +1,17 @@
+package com.univoice.data.repositoryimpl
+
+import com.univoice.data.datasource.UniversityDataSource
+import com.univoice.data.dto.response.ResponseUniversityDto
+import com.univoice.domain.repository.UniversityRepository
+import timber.log.Timber
+import javax.inject.Inject
+
+class UniversityRepositoryImpl @Inject constructor(
+    private val universityDataSource: UniversityDataSource
+) : UniversityRepository {
+    override suspend fun getUniversityNames(): Result<List<String>> {
```

Comment:
P1 : 저는 보통 method + url로 짓는 편
```suggestion
    override suspend fun postUniversity(): Result<List<String>> {
```

Reply:
반영완!